notebook.ipynb

```diff
@@ -11,11 +11,11 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    IS_GOOGLE_CLOUD_NOTEBOOK = os.path.exists("/opt/otron/deeplearning/metadata/env_version")
</pre><p>are you sure about this patH?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/smith-kyle/deep-learning-with-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>No</p>